tests/integrations/async-validation.spec.ts

```diff
@@ -74,7 +74,11 @@ async function runTest(page: Page, javaScriptEnabled: boolean) {
 		'Title is required',
 	]);
 
-	await fieldset.email.press('Control+a');
+	if (process.platform === 'darwin') {
+		await fieldset.email.press('Meta+a');
+	} else {
+		await fieldset.email.press('Control+a');
+	}
```

Comment:
Maybe we can select the whole field with a [double click](https://playwright.dev/docs/api/class-locator#locator-dblclick)? Then we don't need OS specific approach.

Reply:
@edmundhung there are a few other cases where you copy/paste values as well, so I opt-ed for helper functions instead